gutenberg.php

```diff
@@ -260,6 +260,12 @@ function gutenberg_add_edit_link( $actions, $post ) {
 	if ( 'wp_block' === $post->post_type ) {
 		unset( $actions['edit'] );
 		unset( $actions['inline hide-if-no-js'] );
+		$actions['export'] = sprintf(
+			'<a class="wp-list-reusable-blocks__export" href="#" data-id="%s" aria-label="%s">%s</a>',
+			$post->ID,
+			__( 'Export as JSON', 'gutenberg' ),
```

Comment:
Is this convention lifted from somewhere? Doesn't seem obvious why we'd need an `aria-label` which has the same text as the element itself.

Reply:
yes, it's inspired by the way we add the "Classic editor" link in the post list actions